hudi-common/src/main/avro/HoodieCommitMetadata.avsc

```diff
@@ -129,6 +129,11 @@
          }],
          "default": null
       },
+      {
+         "name":"operateType",
+         "type":["null","string"],
+         "default": null
```

Comment:
Also, Can you confirm if the operation type is stored in the avro objects when archiving ?